### PR TITLE
docs: features

### PR DESCRIPTION
**Pull request by vltansky**
Do you think a description necessary for each bullet?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,15 +48,23 @@ Swiper is not compatible with all platforms, it is a modern touch slider which i
...
-- **And many more ...**: All Swiper well-known features are also here: Responsive, Scroll prevention, Resistant bounds, Autoplay, Loop mode, Nested Swipers
+- **Loop mode**
+- **Autoplay**
+- **Keyboard control**
+- **Mousewheel control**
+- **Nested sliders**
+- **History navigation**
+- **Hash navigation**
+- **Breakpoints configuration**
+- **A11y**:
```
Comment (nolimits4web) [inline README.md:66]: Maybe this point can be renamed to `Accessibility (A11y)`?

**Comment by nolimits4web** [comment]
> Do you think a description necessary for each bullet?

Not sure, they are pretty self explaining

File: README.md (hunk 1 above)
Comment (vltansky) [reply README.md]: Done

**Comment by nolimits4web** [comment]
Thanks!

**Pull request merged**